docker/docker-compose.yml

```diff
@@ -3,7 +3,16 @@ version: '2'
 services:
   conductor-server:
     environment:
-      - CONFIG_PROP=config.properties
+      - db=dynomite
+      - workflow_dynomite_cluster_hosts=dyno1:8102:us-east-1c
+      - workflow_dynomite_cluster_name=dyno1
+      - workflow_namespace_prefix=conductor
+      - workflow_namespace_queue_prefix=conductor_queues
+      - queues_dynomite_threads=10
+      - queues_dynomite_nonQuorum_port=22122
+      - workflow_elasticsearch_url=es:9300
+      - workflow_elasticsearch_index_name=conductor
```

Comment:
>  if you run kibana:5.2 and set the ELASTICSEARCH_URL environment variable, I'm pretty sure that it actually writes that value into the kibana.yml as elasticsearch.url.

Yep this is exactly what I mean. I think, put simpler is:

`UPPERCASE_AND_UNDERSCORED_ENV_VAR` should write to `some.internally.used.property`

If this is the case here, feel free to ignore. I might be misunderstanding something with docker-compose and `environment`